pkgs/development/python-modules/mpv/default.nix

```diff
@@ -16,10 +16,10 @@ buildPythonPackage rec {
 
   buildInputs = [ mpv ];
 
-  postPatch = ''
+  patchPhase = ''
```

Comment:
```suggestion
  postPatch = ''
```

Reply:
`postPatch` wasn't applying correctly when included in Sublime Music (I inspected the `mpv.py` file manually in the `/nix` store and it wasn't correctly updated...)

Reply:
postPatch is run in patchPhase when it isn't overwriten. There shouldn't be any difference.